m/micrometer-micrometer-observation/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "santhoshkotla2929",
+    "package_name": "micrometer:micrometer-observation",
```

Comment:
Package name can be micrometer-observation. Accordingly update wherever applicable - script name, script header comment, folder name.